### DISCUSSION TITLE
Princesas Disney Modernas

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#431] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/princesas-disney-modernas/

**Comment by angellisy** [comment]
Talento extraordinário! Das seis escolhidas pra capa não sei dizer qual ficou mais bonita :heart\_eyes:

**Comment by brunoaugusto** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [terça-feira, 31 de maio de 2022](<https://www.vish.blog.br/imagens/princesas-disney-modernas/#comment_1142235658>):

> Talento extraordinário! Das seis escolhidas pra capa não sei dizer qual ficou mais bonita 😍

Eu sou um crianção pra desenhos, devo ter assistido quase todos da Disney, Pixar e cia, mas, dessa lista, acho que eu escolheria a Ariel ou a Anastasia, mesmo que as originais não sejam minhas preferidas.
